### DISCUSSION TITLE
In scheduled reminders for teams, I want to target the team alone, not individual members

### DISCUSSION THREAD

**Discussion by mdub** [community/community#70430] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I set up a [scheduled reminder for my team](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team#creating-a-scheduled-reminder-for-a-team), into a Slack channel. I enabled options like:
* Ignore drafts
* Require review requests

I _wanted_ (and _expected_) to get reminders for review requests where my team had been explicitly tagged for review.

But, when the scheduled reminder came through, it included more than that. After chatting to you Copilot support bot, it seems that scheduled reminders include:

> ... pull requests where a review is specifically requested from the team **or a team member**

I do not want the last (bolded) bit. Please could you add an option allowing reminders to exclude requests targeted at individual team members?

**Comment by zhimsel** [comment]
I'm noticing the same thing and would very much appreciate this option.

To be honest, it's kind of pointless to have a team be notified if only a single member of the team was requested for review. For example, consider a team with members that are on many different teams for different purposes. A scheduled reminder would include reviews that have nothing to do with the team in question because its members belong to many different teams.

**Comment by ndench** [comment]
I think both options are valuable. Depends on the size of the teams and they way they work.

**Reply by zhimsel** [reply]
That's true. I wouldn't want the current behavior to necessarily be _removed_, but at least make it configurable.

**Comment by leedongwei** [comment]
Just wanted to add my voice here:

> where a review is specifically requested from the team or a team member

This doesn't work for a cross-functional project team. We just want PR reminders specific to this team in our project's messaging channel, but we're getting PR reminders for unrelated projects across the company wherever someone on the team is tagged for a review.

It'll be good to have options to configure for both, as discussed above.

**Comment by greenkey** [comment]
Our team is also encountering this issue.

Has anyone found a workaround or alternative solution to ensure that reminders are limited to team-specific review requests? Any insights or suggestions would be greatly appreciated.

Thank you!

**Comment by jackkoppa** [comment]
Adding a +1 for our team. We rely on the many other GitHub integrations for notifying _individuals_ about their open PRs (e.g. the general GitHub Slack bot/reminders). 

We would like the _team_ reminder to notify only for things that anyone from the team has the ability to take action on, and would love to enable that option if it's made available.
